scripts/gen_layer.py

```diff
@@ -0,0 +1,3712 @@
+#!/usr/bin/python3
+#
+# Copyright (c) 2021-2022 LunarG, Inc.
```

Comment:
For any generated file, we typically include a comment at the very top looking something like `*** THIS FILE IS GENERATED - DO NOT EDIT ***` followed by  ` See scripts/gen_layer.py for modifications`.